scripts/npm/create-component.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+/* eslint-disable no-console */
+
+const { execSync } = require('child_process');
+const prompt = require('prompt');
+const fs = require('fs');
+const colors = require('colors');
+
+const STORYBOOK_CONFIG_SPLIT_POINT_1 = 'configure(() => {';
+const STORYBOOK_CONFIG_SPLIT_POINT_2 = '}, module);';
+
+const schema = {
+  properties: {
+    platform: {
+      description: 'native or web?',
+      default: 'native',
+      pattern: /^(native|web)$/,
+      message: 'Enter "native" or "web"',
+    },
+    name: {
+      description: "Enter the component name, e.g. 'banner-alert'",
+      pattern: /^[a-z-]+$/,
+      message: 'Use snake case, e.g. "banner-alert".',
+      required: true,
+    },
+  },
+};
+
+const snakeCaseToPascalCase = input => {
```

Comment:
we have lodash installed for this you can use something like
```
_.mixin({ 'pascalCase': _.flow(_.camelCase, _.upperFirst) });

console.log(_.pascalCase('this-is-snake-case'))
```

Reply:
Ooh that looks cool